bindings/python/crocoddyl/multibody/costs/pair-collisions.cpp

```diff
@@ -0,0 +1,44 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
Remove this cost function (hpp, and bindings).

I understand you have created a deprecated code, however, we don't need to deprecate a class that never was introduced in the library.